test/Grpc.Net.Client.Tests/GrpcChannelTests.cs

```diff
@@ -61,14 +55,11 @@ public void Build_SslCredentialsWithHttp_ThrowsError()
         [Test]
         public void Build_SslCredentialsWithArgs_ThrowsError()
         {
-            // Arrange
-            var httpClient = new HttpClient();
-            httpClient.BaseAddress = new Uri("https://localhost");
-            var channelBuilder = ChannelBuilder.ForHttpClient(httpClient);
-            channelBuilder.SetCredentials(new SslCredentials("rootCertificates!!!"));
-
-            // Assert
-            var ex = Assert.Throws<InvalidOperationException>(() => channelBuilder.Build());
+            // Arrange & Assert
+            var ex = Assert.Throws<InvalidOperationException>(() => GrpcChannel.ForAddress("https://localhost", new GrpcChannelOptions
+            {
+                Credentials = new SslCredentials("rootCertificates!!!")
+            }));
 
             // Act
             Assert.AreEqual("Using SslCredentials with non-null arguments is not supported by GrpcChannel.", ex.Message);
```

Comment:
@JamesNK can you explain why is it not supported? how can one pass a ca-certificate? does it automatically looks at the system ca store? Thanks.

Reply:
Correct. Specifying manually isn’t supported or needed with http client

Reply:
Thanks for your reply. Last question,

When creating instances of `Channel` I could pass [`ChannelOption`](https://github.com/grpc/grpc/blob/master/src/csharp/Grpc.Core/ChannelOptions.cs#L60) and specify load-balancing policy.

```
...
var channelOptions = new List<ChannelOption>
{
    new ChannelOption("grpc.lb_policy_name", "round_robin")
};
var channel = new Channel("https://example.com", chCreds, channelOptions);
```

but now when registering gRPC client using GrpcClientFactory I don't see any option to pass these to [`GrpcChannelOption`](https://github.com/grpc/grpc-dotnet/blob/master/src/Grpc.Net.Client/GrpcChannelOptions.cs#L26). Is it not supported?

Reply:
Those options? No. That's a different gRPC client implementation.